docs/release-process.md

```diff
@@ -52,8 +52,8 @@ This only needs to be done once.
    This guide uses your Github username as the fork remote name.
 
    ```
-   git remote add {USERNAME} https://github.com/{USERNAME}/ansible-build-data
-   git fetch {USERNAME} -v
+   git remote add ${USERNAME} https://github.com/${USERNAME}/ansible-build-data
+   git fetch ${USERNAME} -v
    ```
 
 ## Preform release process
```

Comment:
```suggestion
## Perform release process
```

While your at it, maybe you could also fix this typo :-)

At least, I think that it's a typo and "perform" is meant here.

Reply:
Indeed!